medusa/network_timezones.py

```diff
@@ -67,17 +67,26 @@ def update_network_dict():
     network_list = dict(cache_db_con.select('SELECT * FROM network_timezones;'))
 
     queries = []
-    for network, timezone in iteritems(d):
+    remote_network_list = iteritems(d)
```

Comment:
Sure.
Its not a concluded PR. I still cant reproduce the issue
Point is to discuss solutions.
Thanks